### DISCUSSION TITLE
"dashboard panel drop down > explore" cannot open in new browser window

### DISCUSSION THREAD

**Discussion by max-rocket-internet** [grafana/grafana#38219] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Currently the dashboard panel drop down menu option `Explore` cannot be opened in a new browser tab/window due to it not being a proper link:

 ![Screenshot 2021-02-12 at 12 07 59](https://user-images.githubusercontent.com/8859277/107761102-1ba8ca00-6d2b-11eb-8f05-91088a4f4bc0.png)

You can see here that when you right-click, the context menu is the normal text menu, not a link menu:

![Screenshot 2021-02-12 at 12 05 20](https://user-images.githubusercontent.com/8859277/107761200-4430c400-6d2b-11eb-8777-4368a1b7e966.png)

**Why is this needed**:

I know it sounds minor but if you're working with Grafana and prometheus a lot and jumping between dashboards and queries, it's really annoying having to lose your dashboard page in order to debug a panel. I am always having to duplicate tabs just to select `Explore` on the new tab and not lose my current dashboard page.

**Comment by max-rocket-internet** [comment]
On Mac I can do `command + click` to open new window but middle click does not work and neither does right click. I'm not sure how it works on Windows.

**Comment by peterbourgon** [comment]
This is a consistent source of frustration for me :(

Can the default behavior be made configurable somehow? I essentially never want Explore to open in the same tab.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
